matxmat44_neon.c

```diff
@@ -56,7 +55,11 @@ void matXmat_neon(float *A[], float B[N][N], float *D[]) {
   }
 }
 
-int main() {
+int main(int argc, char** argv) {
+  int LOOPS = 100;
```

Comment:
if you're going to do that, set a default with a #define, but you also need to do error handling, eg. if the 2nd argument is not a number and atoi() fails, then LOOPS will be set to an undefined value.
But 100 is a very very small number, I'd put something that gives a reasonable result in eg. ~1sec in most systems. But good incentive.

Reply:
Yes, will fix everything you mentioned. I just did the first commit to create the PR. Should have written WIP.